keyboards/system76/launch_1/launch_1.c

```diff
@@ -0,0 +1,238 @@
+/* 3 */ 38, 37, 36, 35, 34, 33, 32, 31, 30, 29, 28, 27, 26,     25,
+/* 4 */ 12, 11, 10,  9,  8,  7,  6,  5,  4,  3,  2,  1,  0,
+/* 5 */ 13, 14, 15, 16, 17,     18, 19, 20, 21, 22, 23, 24
+), {
+    // LED index to physical position (see leds.sh in `launch' repo)
+/* 00 */ {209, 51}, {190, 51}, {171, 51}, {156, 51}, {140, 51}, {125, 51}, {110, 51}, {95, 51}, {80, 51}, {65, 51},
+/* 10 */ {49, 51}, {34, 51}, {11, 51}, {8, 64}, {27, 64}, {42, 64}, {57, 64}, {80, 64}, {110, 64}, {133, 64},
+/* 20 */ {148, 64}, {167, 64}, {194, 64}, {209, 64}, {224, 64}, {224, 38}, {197, 38}, {178, 38}, {163, 38}, {148, 38},
+/* 30 */ {133, 38}, {118, 38}, {103, 38}, {87, 38}, {72, 38}, {57, 38}, {42, 38}, {27, 38}, {8, 38}, {4, 26},
+/* 40 */ {23, 26}, {38, 26}, {53, 26}, {68, 26}, {84, 26}, {99, 26}, {114, 26}, {129, 26}, {144, 26}, {159, 26},
+/* 50 */ {175, 26}, {190, 26}, {205, 26}, {224, 26}, {224, 13}, {201, 13}, {182, 13}, {167, 13}, {152, 13}, {137, 13},
+/* 60 */ {121, 13}, {106, 13}, {91, 13}, {76, 13}, {61, 13}, {46, 13}, {30, 13}, {15, 13}, {0, 13}, {0, 0},
+/* 70 */ {15, 0}, {30, 0}, {46, 0}, {61, 0}, {76, 0}, {91, 0}, {106, 0}, {121, 0}, {137, 0}, {152, 0},
+/* 80 */ {167, 0}, {182, 0}, {201, 0}, {224, 0}
+}, {
+    // LED index to flags (set all to LED_FLAG_KEYLIGHT)
+    /*   0  1  2  3  4  5  6  7  8  9 */
+/* 00 */ 4, 4, 4, 4, 4, 4, 4, 4, 4, 4,
+/* 10 */ 4, 4, 4, 4, 4, 4, 4, 4, 4, 4,
+/* 20 */ 4, 4, 4, 4, 4, 4, 4, 4, 4, 4,
+/* 30 */ 4, 4, 4, 4, 4, 4, 4, 4, 4, 4,
+/* 40 */ 4, 4, 4, 4, 4, 4, 4, 4, 4, 4,
+/* 50 */ 4, 4, 4, 4, 4, 4, 4, 4, 4, 4,
+/* 60 */ 4, 4, 4, 4, 4, 4, 4, 4, 4, 4,
+/* 70 */ 4, 4, 4, 4, 4, 4, 4, 4, 4, 4,
+/* 80 */ 4, 4, 4, 4
+} };
+#endif // RGB_MATRIX_ENABLE
+
+bool eeprom_is_valid(void) { 
+    return (
+        eeprom_read_word(((void *)EEPROM_MAGIC_ADDR)) == EEPROM_MAGIC &&
+        eeprom_read_byte(((void *)EEPROM_VERSION_ADDR)) == EEPROM_VERSION
+    );
+}
+// clang-format on
+
+void eeprom_set_valid(bool valid) {
+    eeprom_update_word(((void *)EEPROM_MAGIC_ADDR), valid ? EEPROM_MAGIC : 0xFFFF);
+    eeprom_update_byte(((void *)EEPROM_VERSION_ADDR), valid ? EEPROM_VERSION : 0xFF);
+}
+
+void eeprom_reset(void) {
+    // Set the keyboard-specific EEPROM state as invalid
+    eeprom_set_valid(false);
+    // Set the TMK/QMK EEPROM state as invalid
+    eeconfig_disable();
+}
+
+// The lite version of TMK's bootmagic based on Wilba.
+// 100% less potential for accidentally making the keyboard do stupid things.
+void bootmagic_lite(void) {
+    // Do multiple scans because debouncing cannot be turned off
+    matrix_scan();
+#if defined(DEBOUNCE) && DEBOUNCE > 0
+    wait_ms(DEBOUNCE * 2);
+#else
+    wait_ms(30);
+#endif
+    matrix_scan();
+
+    // If the `Esc' key (matrix 0,0) is held down on power up,
+    // reset the EEPROM valid state and jump to bootloader
+    if (matrix_get_row(0) & (1 << 0)) {
+        eeprom_reset();
+        bootloader_jump();
+    }
+}
```

Comment:
This is defined by default. 
```suggestion
```

Reply:
Change results in not being able to enter the bootloader, requiring electrical reset to flash firmware. There must be some inconsistency somewhere. To be specific these are the changes that caused the issue:
- Remove the `bootmagic_lite` function as it is already defined in `bootmagic_lite.c`
- No need to define `BOOTMAGIC_LITE_ROW/COLUMN` since existing function uses zeros
- Rename `eeprom_reset` function to `bootmagic_lite_reset_eeprom` so that the version in `bootmagic_lite.c` gets replaced
- Remove `bootmagic_lite` call from `matrix_init_kb` function
- Set `BOOTMAGIC_ENABLE = yes` in `rules.mk`

Might revert the changes proposed in this suggestion, pending some other insight.

Reply:
Tested several variations on the production board. There is something preventing the `BOOTMAGIC_ENABLE = yes` from functioning as expected. So reverted the change back to the duplicated version until the underlying issue can be root caused.